src/sql/src/plan/lowering/variadic_left.rs

```diff
@@ -364,6 +378,7 @@ pub(crate) fn attempt_left_join_magic(
     Ok(Some(body))
 }
 
+use crate::optimizer_metrics::OptimizerMetrics;
```

Comment:
The place for this import seems strange, given that the imported type is only used further up in this file.

Reply:
fixed